src/rules/prBodyNotEmpty.ts

```diff
@@ -0,0 +1,79 @@
+import type { Rule } from "../types/rules.js";
+
+import { wrapSafe } from "../types/utils.js";
+
+export const prBodyNotEmpty = {
+	about: {
+		config: "recommended",
+		description: "PR should have a description beyond the template.",
```

Comment:
[Docs] Nit: the other ones go with plural for PRs

```suggestion
		description: "PRs should have a description beyond the template.",
```